controller/src/main/java/uni/miskolc/ips/ilona/tracking/controller/MobileRequestsController.java

```diff
@@ -45,6 +48,7 @@ public MobileTransferDTO mobileSendMeasurementHandler(@RequestBody(required = fa
 		
 		UserSecurityDetails userDetails = (UserSecurityDetails) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
 		System.out.println(userDetails.toString());
```

Comment:
The userDetails is why printed out? What is the purpose of this command?

Reply:
I don't know, I did't wrote it